onnxruntime/python/tools/transformers/models/whisper/benchmark.py

```diff
@@ -352,6 +360,13 @@ def without_io_binding(inputs):
         outputs = model.run(None, inputs)
         return outputs
 
+    def handle_output(output):
+        if 50257 in output:
+            first_end = np.where(output == 50257)[0][0]
+            return output[: first_end + 1]
```

Comment:
The end-of-sequence token id is different for the English-only models (e.g. `openai/whisper-tiny.en`). We can add `setattr(args, "eos_token_id", config.eos_token_id)  # noqa: B010` after these lines.

https://github.com/microsoft/onnxruntime/blob/770531d6dc6170fefb29f4e116e88cf4834b9ef0/onnxruntime/python/tools/transformers/models/whisper/benchmark.py#L553-L561

Then we can replace `50257` in `handle_output` with `args.eos_token_id`.